# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/util/ChromeProcessController.cpp
# Commit: 024cf173d87d
# Full Hash: 024cf173d87dbc5d3b581d2b121e3d8295e4f46f
# Author: Dan Robertson <drobertson@mozilla.com>
# Date: 2023-09-26 09:27:49
# Regressor Bug: 1816473
# File Overlap Count: 3
# Description:
#   Bug 1816473 - Do not delay dispatching synthesized mouse events. r=botond,edgar
#   
#   Do not delay dispatching synthesized mouse events for a single tap
#   gesture.
#   
# ==============================================================================

diff -r 37eee8ccc556 -r 024cf173d87d gfx/layers/apz/util/ChromeProcessController.cpp
--- a/gfx/layers/apz/util/ChromeProcessController.cpp	Mon Sep 25 19:08:05 2023 +0000
+++ b/gfx/layers/apz/util/ChromeProcessController.cpp	Mon Sep 25 19:13:07 2023 +0000
@@ -94,6 +94,9 @@
 
   MOZ_ASSERT(mUIThread->IsOnCurrentThread());
   mWidget = nullptr;
+  if (mAPZEventState) {
+    mAPZEventState->Destroy();
+  }
   mAPZEventState = nullptr;
 }
 
